aidial_assistant/commands/run_plugin.py

```diff
@@ -33,6 +33,7 @@
 
 class PluginInfo(BaseModel):
     info: OpenAIPluginInfo
+    name_override: str | None
```

Comment:
`name_override` implies that a certain name will be overriden by this one. But what name would it be? Is it `name_for_model` or `name_for_human`? And why not simply override the one of those inplace in `OpenAIPluginInfo`? If it wasn't done, then it means that all three names are important. So it's better to give them the naming telling **what they are**, instead of **what they do** to avoid confusion.

E.g. `name_from_request`.

Reply:
Removed the property from the PluginInfo class.